hack/boilerplate.go.txt

```diff
@@ -1,7 +1,5 @@
-// +build !ignore_autogenerated
-
 /*
-Copyright The Kubernetes Authors.
+
 
```

Comment:
Extra space?

Reply:
mm I didn't edit this :/ ok I'll remove it.